hw-model/src/model_fpga_realtime.rs

```diff
@@ -19,21 +19,20 @@ use crate::Output;
 const SOC_PAUSER: u32 = 0xffff_ffff;
 
 // UIO mapping indices
-const GPIO_MAPPING: usize = 0;
-const MBOX_MAPPING: usize = 1;
-const SOC_IFC_MAPPING: usize = 2;
+const SOC_MAPPING: usize = 0;
```

Comment:
"SOC" is too confusing. I should change this (and other re-names) to something like "FPGA_WRAPPER" instead.

Reply:
CALIPTRA_APB_MAPPING?